test/SlowTests/Issues/RavenDB-23892.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Operations.Backups;
+using Raven.Client.Documents.Operations.OngoingTasks;
+using Raven.Client.ServerWide.Operations;
+using Raven.Client.ServerWide.Operations.Configuration;
+using Raven.Client.ServerWide.Operations.OngoingTasks;
+using SlowTests.Core.Utils.Entities;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_23892 : RavenTestBase
+    {
+        public RavenDB_23892(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [RavenFact(RavenTestCategory.BackupExportImport)]
+        public async Task Snapshot_Should_Not_Include_ServerWideBackupConfiguration()
+        {
+            using (var server = GetNewServer())
+            using (var store = GetDocumentStore(new Options { Server = server }))
+            {
+                var database = store.Database;
+                using (var session = store.OpenAsyncSession())
+                {
+                    await session.StoreAsync(new User { Name = "Toli" }, "users/1");
+                    await session.SaveChangesAsync();
+                }
+                var serverWideConfig = new ServerWideBackupConfiguration
+                {
+                    Name = "ServerWideBackup",
+                    Disabled = false,
+                    BackupType = BackupType.Backup,
+                    FullBackupFrequency = "0 1 * * *",
+                    LocalSettings = new LocalSettings { FolderPath = "test/folder" }
+                };
+                var result = await store.Maintenance.Server.SendAsync(new PutServerWideBackupConfigurationOperation(serverWideConfig));
```

Comment:
how about adding a convention tests, that will fail if any new Client-Operation and Cluster-Command is added that contains 'ServerWide' in the name? This way, if something new will be added (e.g. Server-Wide Connection Strings) then this test will fail automatically and point developer (assuming that proper comment will be added with good description) to a place where it should be fixed. Thoughts?

Reply:
Done